recipes/eva-sub-cli/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+VCF_VALIDATOR_VERSION=0.9.6
+BIOVALIDATOR_VERSION=2.1.0
+EVA_PYUTILS_VERSION=0.6.1
+
+EVA_SUB_CLI="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}"
+mkdir -p ${PREFIX}/bin ${EVA_SUB_CLI}
+
+# Install eva-sub-cli
+$PYTHON -m pip install .
+cp bin/* ${PREFIX}/bin
+echo "Done with eva-sub-cli"
+
+cd ${EVA_SUB_CLI}
+
+# Install python dependencies not yet on conda
+curl -Lo eva-pyutils.zip https://github.com/EBIvariation/eva-common-pyutils/archive/refs/tags/v${EVA_PYUTILS_VERSION}.zip \
+    && unzip -q eva-pyutils.zip && rm eva-pyutils.zip \
+    && cd eva-common-pyutils-${EVA_PYUTILS_VERSION} \
+    && $PYTHON setup.py install \
+    && cd ..
+
+# Install biovalidator from source
+# Includes some workarounds that can be cleaned up once a new version is released
+curl -Lo biovalidator.zip https://github.com/elixir-europe/biovalidator/archive/refs/tags/v${BIOVALIDATOR_VERSION}.zip \
```

Comment:
Should we then wait for a new release?

Reply:
Yes, we can wait for a new tagged release. I've also requested a [conda recipe](https://github.com/elixir-europe/biovalidator/issues/73) from the maintainers, in case that's something they're willing to do.

Reply:
updated to use newest version